src/legacy/ui/ui_bundles/app_entry_template.js

```diff
@@ -33,6 +33,7 @@ import 'dll/set_csp_nonce';
 __webpack_nonce__ = window.__kbnNonce__;
 
 // import global polyfills
+import Symbol_observable from 'symbol-observable';
```

Comment:
Can you add this to `src/legacy/core_plugins/tests_bundle/tests_entry_template.js` too please?